src/main/java/io/github/wimdeblauwe/hsbt/mvc/HxRefresh.java

```diff
@@ -0,0 +1,12 @@
+package io.github.wimdeblauwe.hsbt.mvc;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({ElementType.TYPE, ElementType.METHOD})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface HxRefresh {
```

Comment:
This looks good and I'm fine merging this as is. 

However, I've got a nagging feeling that `hx-push`, `hx-redirect`, and `hx-refresh` are all related. They trigger browser URL/page behavior. This isn't a proposal, but I wonder if there is a `@HxPage(pushUrl="", redirect="", refresh="") commonality. I can't imagine a case where the user would need multiple, but I wanted to point out this grouping.

Reply:
ok, we will have to see how users react to how it is now I guess. If there is a need, we can always change it. We are not near 1.0 just yet 😄 But it is good that you add these comments, other people can react to it if they feel the same way.